src/arch/arm/64/object/objecttype.c

```diff
@@ -415,41 +341,23 @@ cap_t Arch_createObject(object_t t, void *regionBase, word_t userSize, bool_t de
                    VMReadWrite,           /* capFVMRights */
                    !!deviceMemory         /* capFIsDevice */
                );
-#ifndef AARCH64_VSPACE_S2_START_L1
-    case seL4_ARM_PageGlobalDirectoryObject:
+    case seL4_ARM_VSpaceObject:
 #ifdef CONFIG_ARM_SMMU
 
-        return cap_page_global_directory_cap_new(
+        return cap_vspace_cap_new(
                    asidInvalid,           /* capPGDMappedASID   */
                    (word_t)regionBase,    /* capPGDBasePtr      */
                    0,                     /* capPGDIsMapped     */
                    CB_INVALID             /* capPGDMappedCB     */
                );
 #else
 
-        return cap_page_global_directory_cap_new(
+        return cap_vspace_cap_new(
                    asidInvalid,           /* capPGDMappedASID   */
                    (word_t)regionBase,    /* capPGDBasePtr      */
                    0                      /* capPGDIsMapped     */
                );
 #endif /*!CONFIG_ARM_SMMU*/
```

Comment:
the comments in this bit still are referring to `capPGD`. Should probably update.